ndn-svs/svspubsub.hpp

```diff
@@ -56,16 +56,37 @@ class SVSPubSub : noncopyable
   virtual
   ~SVSPubSub() = default;
 
-  /** Subscription Data type */
-  struct SubscriptionPacket
+  struct SubscriptionData
   {
-    const Data& data;
+    /** @brief The name of the received publication */
+    const Name& name;
+
+    /** @brief Raw buffer of the received data */
+    const uint8_t* data;
+
+    /** @brief Length of the received data */
+    const size_t length;
```

Comment:
why do you need these two fields if you're already keeping a reference to the content block?

Reply:
This was somewhat sloppy. Changed everything to exclusively use span now.